Kudu.Services.Test/Jobs/InvokeTriggeredJobReadonlyFileSystemTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Kudu.Contracts.Jobs;
+using Kudu.Contracts.Tracing;
+using Kudu.Core.Infrastructure;
+using Kudu.Services.Jobs;
+using Kudu.TestHarness.Xunit;
+using Moq;
+using System;
+using System.IO.Abstractions;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+
+namespace Kudu.Services.Test.Jobs
+{
+    [KuduXunitTestClass]
+    public class InvokeTriggeredJobReadonlyFileSystemTests
```

Comment:
Since this test is testing a controller method of JobsController, I think it should be moved to an existing JobController  test class (hopefully one exists), otherwise create a new test class for JobsController. Then, the test method should be named something like InvokeTriggeredJob_ReturnsServiceUnavailable_WhenFileSystemIsReadOnly